packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2758,8 +2758,13 @@ export interface VideoGalleryProps {
 export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
     isSpeaking?: boolean;
     screenShareStream?: VideoGalleryStream;
+    // @beta
+    state?: VideoGalleryRemoteParticipantState;
 }
 
+// @beta
+export type VideoGalleryRemoteParticipantState = 'Connecting' | 'Ringing' | 'Hold';
```

Comment:
Should something like 'connected' or 'inCall' be a state?

Reply:
Didn't include those states because we don't show the state in our video tile visually when the participant is incall or connected.

Reply:
My thinking is that it does impact the UI - by explicitly not showing a state in the video tile. Contoso should be always able to set all props and an enum/type like `VideoGalleryRemoteParticipantState` should show all the possibilities - if I tried to use that prop its not clear that I can set it as undefined and get a different behavior than if I set it to one of the string litreral values.

Reply:
The concern here is by adding `Connected` to the enum, we are suggesting that similar to `Ringing...` and `Connecting...` text under the Persona, `Connected` will also be shown. Which is misleading.

Reply:
I agree that `Connected` should be included.
We would expect the state to be `Connected` for participants actually in the call (once our selectors have been updated) so we should leave a note that `undefined` state is treated the same as `Connected` for backward compatibility.

Reply:
I don't think the value of `Connected` is misleading. The field isn't called `connectionOverlayLabel` or some such, in which case it would be misleading to show nothing for some explicit value.

An alternative implementation of the `VideoTile` might not even attempt to show the video until state enters `Connected`...